Code/GraphMol/DistGeomHelpers/Embedder.cpp

```diff
@@ -721,42 +756,91 @@ bool embedPoints(RDGeom::PointPtrVect *positions, detail::EmbedArgs eargs,
       std::cerr << "Initial embedding failed!, Iter: " << iter << std::endl;
```

Comment:
This is within #ifdef DEBUG_EMBEDDING, but it is still two "if (!gotCoords) {}" statements after each other (line 720 and 724). Can they maybe be combined?

Reply:
Even simpler: I just removed the debugging statement.